hub/models.py

```diff
@@ -2246,7 +2270,7 @@ async def update_many(self, mapped_records, **kwargs):
             ]
         )
 
-    def auto_update_webhook_specification(self):
+    def auto_webhook_specification(self):
         if self.geography_column is None:
             raise ValueError("A geography column is required for auto-updates to work.")
```

Comment:
This should only be relevant to auto-updates. Imports should probably happen even for non-geographical data.

Reply:
If we are re-using the same webhook for import and update, does that mean we always have to listen to changes on any field?